### PR TITLE
cleanup optocoupler

### PR DESCRIPTION
**Pull request by LoQue90**
cleanup optocoupler to follow camelcase code style

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -120,18 +120,18 @@ const unsigned long fillTime = FILLTIME;
 const unsigned long flushTime = FLUSHTIME;
 int maxflushCycles = MAXFLUSHCYCLES;
 
-// Voltage Sensor
-unsigned long previousMillisVoltagesensorreading = millis();
-const unsigned long intervalVoltagesensor = 200;
-int optoCouplerON, optoCouplerOFF;
+// Optocoupler
+unsigned long previousMillisOptocouplerreading = millis();
```
Comment (kjyv) [inline src/main.cpp:124]:
```suggestion
unsigned long previousMillisOptocouplerReading = millis();
```

File: src/main.cpp [hunk 2]
```diff
@@ -825,9 +825,9 @@ void brewDetection() {
 
             // no Quickmill:
             default:
-                previousMillisVoltagesensorreading = millis();
+                previousMillisOptocouplerreading = millis();
```
Comment (kjyv) [inline src/main.cpp:828]:
```suggestion
                previousMillisOptocouplerReading = millis();
```

File: src/main.cpp [hunk 3]
```diff
@@ -2131,7 +2131,7 @@ void setup() {
     previousMillisDisplay = currentTime;
     previousMillisMQTT = currentTime;
     previousMillisInflux = currentTime;
-    previousMillisVoltagesensorreading = currentTime;
+    previousMillisOptocouplerreading = currentTime;
```
Comment (kjyv) [inline src/main.cpp:2134]:
```suggestion
    previousMillisOptocouplerReading = currentTime;
```

Comment (LoQue90) [reply src/main.cpp:2134]: Wasn't sure about that because everywhere I saw a var with reading at the end was writen small.
Going to change it, thanks!

Comment (kjyv) [reply src/main.cpp:2134]: I think the rule of thumb is that we want to have a capital letter whenever a new word starts

**Review by kjyv — DISMISSED** [review]

**Comment by LoQue90** [comment]
sorry @kjyv, had to do a rebase one more time, now you should be able to merge this, thank you!

**Review by kjyv — APPROVED** [review]

**Pull request merged**